image/image_linux.go

```diff
@@ -636,5 +646,13 @@ var setupSeed = func(tsto *tooling.ToolingStore, model *asserts.Model, opts *Opt
 		customizeImage(rootDir, defaultsDir, &opts.Customizations)
 	}
 
+	// last thing is to generate the image seed manifest file
+	// XXX: is this the correct place to do this? Should we instead
+	// return the "imageManifest" and the let cmd_prepare_image do this?
```

Comment:
the location is ok, another approach would be to return imageManifest and do the writing from Prepare, doing it in prepare-image though seems inappropriate

Reply:
Ok, thanks for confirming this! I'll leave it here